pb/event.go

```diff
@@ -53,6 +55,16 @@ func (e *ReviewEvent) Revision() *CommitRevision {
 	return &e.CommitRevision
 }
 
+// Validate honors the Event interface.
+func (e *ReviewEvent) Validate() error {
+	var zeroVal ReviewEvent
+	if reflect.DeepEqual(*e, zeroVal) {
```

Comment:
you don't need `reflect.DeepEqual`. `*e == zeroVal` should work just fine

Reply:
I tried that first, but fails with:
`invalid operation: *e == zeroVal (struct containing types.Struct cannot be compared)`

Reply:
wow. 👍